addon/components/infinity-loader.js

```diff
@@ -13,6 +13,7 @@ export default Ember.Component.extend({
   destroyOnInfinity: false,
   developmentMode: false,
   scrollable: null,
+  didSendActionFromScrollEvent: false,
```

Comment:
I changed `Ember.run.once(this, this._checkIfInView)` to `Ember.run.scheduleOnce('afterRender', this, this._checkIfInView)` to put it in a different queue, and I no longer need this state.